src/constants/Features.js

```diff
@@ -1,4 +1,4 @@
-const MAPBOX_COORDS = {
```

Comment:
we may want to consider moving these outside of the codebase in order to make it easier for the farm to edit them